tests/docker/irods.ubuntu22.Dockerfile

```diff
@@ -1,44 +1,73 @@
 FROM ubuntu:22.04
 
-ADD start.irods.ubuntu22.sh /
-RUN chmod u+x /start.irods.ubuntu22.sh
-
 ARG DEBIAN_FRONTEND=noninteractive
 ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=true
 
 #### install basic packages ####
 RUN apt-get update && \
-    apt-get install -y apt-utils apt-transport-https unixodbc unixodbc-dev wget lsb-release sudo \
-                       libssl-dev super lsof postgresql odbc-postgresql libjson-perl gnupg \
-                       vim sudo rsyslog g++ dpkg-dev cdbs libcurl4-openssl-dev \
-                       tig git libpam0g-dev libkrb5-dev libfuse-dev \
-                       libbz2-dev libxml2-dev zlib1g-dev \
-                       make gcc help2man telnet ftp
-
-RUN apt-get install -y python3 \
+    apt-get install -y apt-utils \
+    apt-transport-https \
+    unixodbc \
+    unixodbc-dev \
+    wget \
+    lsb-release \
+    sudo \
+    libssl-dev \
+    super \
+    lsof \
+    postgresql \
+    odbc-postgresql \
+    libjson-perl \
+    gnupg \
+    sudo \
+    rsyslog \
+    g++ \
+    dpkg-dev \
+    cdbs \
+    libcurl4-openssl-dev \
+    tig \
+    git \
+    libpam0g-dev \
+    libkrb5-dev \
+    libfuse-dev \
+    libbz2-dev \
+    libxml2-dev \
+    zlib1g-dev \
+    make \
+    gcc \
+    help2man \
```

Comment:
Why are development packages being installed in this docker image?

Reply:
I removed these and other dev packages and tests passed.